frontend/apps/marketing/src/app/layout.tsx

```diff
@@ -1,5 +1,7 @@
 import type {Metadata} from 'next';
 import './globals.css';
+import '@code-dot-org/css/colors.css';
+import '@code-dot-org/css/primitiveColors.css';
```

Comment:
@levadadenys @kelbyhawn - I'm thinking this may be where we would import these color css variables once (rather than our current mechanism where it does it on each module scss).

On `apps` it may need to be a rails import or webpack polyfill on every entrypoint.

Reply:
> On apps it may need to be a rails import or webpack polyfill on every entrypoint.

This is one of the reasons why we implemented it this way from the beginning. We can change this later, but I don't think we need to change it right now, since we have more prioritised tasks I believe.